R/coord-polar.r

```diff
@@ -281,7 +281,7 @@ CoordPolar <- ggproto("CoordPolar", Coord,
     # Combine the two ends of the scale if they are close
     theta <- theta[!is.na(theta)]
     ends_apart <- (theta[length(theta)] - theta[1]) %% (2*pi)
-    if (length(theta) > 0 && ends_apart < 0.05) {
+    if (length(theta) > 0 && ends_apart < 0.05 & !is.null(labels)) {
```

Comment:
I think you want to use `&&` here, not `&`.

Reply:
Ups. Fixed. 
Thanks.